gitlab/.env-dist

```diff
@@ -0,0 +1,11 @@
+GITLAB_TRAEFIK_HOST=git.example.com
+GITLAB_SSH_PORT=2224
+# Choose Let's Encrypt 'staging' or 'production' environment:
+ACME_CERT_RESOLVER=production
+
+POSTGRES_USER=gitlab
+POSTGRES_HOST=postgresql
+POSTGRES_PORT=5432
+POSTGRES_PASS=password
+POSTGRES_DB_NAME=gitlab_production
+GITLAB_ROOT_PASSWORD=changeme
```

Comment:
passwords should be blank in .env-dist, otherwise the Makefile won't randomize it.